tests/integration/routes/account-test.js

```diff
@@ -156,9 +156,9 @@ getServer(function (error, server) {
       server.inject(getAccountRouteOptions, function (response) {
         t.is(couch.pendingMocks()[0], undefined, 'all mocks satisfied')
 
-        t.is(response.statusCode, 404, 'returns 404 status')
+        t.is(response.statusCode, 401, 'returns 401 status')
         t.is(response.result.errors.length, 1, 'returns one error')
-        t.is(response.result.errors[0].title, 'Not Found', 'returns "Not Found" error')
+        t.is(response.result.errors[0].title, 'Unauthorized', 'returns "Unauthorized" error')
```

Comment:
no, sorry :) `.title` is correct, but can you please add a check for `.detail`? `.detail` should be `'Invalid session'`